specification/programmableconnectivity/Azure.ProgrammableConnectivity/apis/location.tsp

```diff
@@ -0,0 +1,81 @@
+import "./common.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+
+namespace Azure.ProgrammableConnectivity;
+
+/// Interfaces
+
+interface LocationInterface {
+  @doc("Verifies whether a device is within a specified location area, defined as an accuracy (radius) around a point, specified by longitude and latitude.")
+  verify is Operations.ResourceAction<
+    LocationEndpoint,
+    LocationVerifyRequest,
+    LocationVerifyResponse
+  >;
+}
+
+/// Endpoints
+
+@resource("DeviceLocation")
+@doc("Static endpoint to access the Device Location API family.")
+model LocationEndpoint {
+  @key
+  @doc("Static endpoint")
+  @visibility("read")
+  location: LocationEnum;
+}
+
+/* Needed to add this enum because typespec was creating an x-ms-enum without a name
+   which was causing a failure during linting. */
+@doc("Location static endpoint enum")
+enum LocationEnum {
+  @doc("Static endpoint")
+  location: "Location",
+}
```

Comment:
Is it an Enum that later could have more values? Or is it a constant that it would NEVER have more values?